### PR TITLE
Initial commit of Arcs manifest tests from original Arcs repo.

### PR DESCRIPTION
**Pull request by markww**
This commit copies over manifest tests from the original Arcs repo.
Right now, all of these tests are in the directory
//src/analysis/souffle/tests/arcs_manifest_tests_todo , a new directory
for manifest tests that we cannot yet handle end to end. As we get these
tests working end to end, we should move them to a new
//src/analysis/souffle/tests/arcs_manifest_tests directory.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -29,9 +29,13 @@ jobs:
...
       # Runs a single command using the runners shell
-      - name: Run a one-line script
+      - name: Run all tests
         run: bazel test //src/...
 
       # Verifies that the Arcs parser works
       - name: Verify that Arcs parser works.
         run: bazel build //third_party/arcs/examples:consume
+
+      # Verifies that Arcs manifest tests that we do not yet handle parse correctly.
```
Comment (bgogul) [inline .github/workflows/main.yml:39]: Just an observation. I think it is time to set up something more performant for tests. It is already taking up to 17m per tests.

File: src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD [hunk 2]
```diff
@@ -0,0 +1,15 @@
+load("//build_defs:arcs.bzl", "arcs_manifest_proto")
+
+# TODO(Issue #61): Enable translating these manifests to proto once manifest2proto can parse them.
+BUILD_ERROR_ARCS_MANIFEST_PATTERNS = [
+    "*fail_field_nullable_inline_entity_direct.arcs",
+    "*ok_join_simple_with_nullables.arcs",
+]
+
+BUILDING_ARCS_MANIFESTS = glob(["*.arcs"], BUILD_ERROR_ARCS_MANIFEST_PATTERNS)
```
Comment (bgogul) [inline src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD:9]: Let us be explicit about the second parameter: 
```suggestion
BUILDING_ARCS_MANIFESTS = glob(
   ["*.arcs"], 
   excludes = BUILD_ERROR_ARCS_MANIFEST_PATTERNS
)
```

File: src/analysis/souffle/tests/arcs_manifest_tests_todo/README [hunk 3]
```diff
@@ -0,0 +1,3 @@
+These tests are imported from the original Arcs repo. They are here because we do not yet correctly
```
Comment (bgogul) [inline src/analysis/souffle/tests/arcs_manifest_tests_todo/README:1]: Can you also mention #64 in the README? There is so much code duplication in the .arcs test files. I wanted to fix this in Arcs, but never got around to it.

**Review by bgogul — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (markww) [reply .github/workflows/main.yml:39]: Acknowledged.

File: src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD (hunk 2 above)
Comment (markww) [reply src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD]: Done.

File: src/analysis/souffle/tests/arcs_manifest_tests_todo/README (hunk 3 above)
Comment (markww) [reply src/analysis/souffle/tests/arcs_manifest_tests_todo/README:1]: Done.

**Pull request merged**